bwmenu

```diff
@@ -158,28 +163,133 @@ clipboard-xsel-clear() {
   xsel --clipboard --delete
 }
 
-# Output the password
+# Copy the password
 # copy to clipboard and give the user feedback that the password is copied
 # $1: json item
-show_password() {
+copy_password() {
   pass=$(echo "$1" | jq -r '.login.password')
 
-  # send a notification for 5 seconds (-t 5000)
-  # not sure if icon will be present everywhere, /usr/share/icons is default icon location
-  notify-send "<b>$(echo "$1" | jq -r '.name')</b> copied" "${pass:0:4}****" -t 5000  -i /usr/share/icons/hicolor/64x64/apps/bitwarden.png
+  show_copy_notification "$1"
   echo -n "$pass" | clipboard-set
 
-  sleep 5
-  if [[ "$(clipboard-get)" == "$pass" ]]; then
-    clipboard-clear
+  if [[ $CLEAR -gt 0 ]]; then
+    sleep "$CLEAR"
+    if [[ "$(clipboard-get)" == "$pass" ]]; then
+      clipboard-clear
+    fi
+  fi
+}
+
+# Show notification about the password being copied.
+# $1: json item
+show_copy_notification() {
+  local title
+  local body=""
+
+  title="<b>$(echo "$1" | jq -r '.name')</b> copied"
+
+  if [[ $SHOW_PASSWORD == "yes" ]]; then
+    pass=$(echo "$1" | jq -r '.login.password')
+    body="${pass:0:4}****"
+  fi
+
+  if [[ $CLEAR -gt 0 ]]; then
+    body="$body<br>Will be cleared in ${CLEAR} seconds."
   fi
+  # send a notification for 5 seconds (-t 5000)
+  # not sure if icon will be present everywhere, /usr/share/icons is default icon location
+  notify-send "$title" "$body" -t 5000  -i /usr/share/icons/hicolor/64x64/apps/bitwarden.png
+}
+
+  while true; do
+    case "$1" in
+      --help )
+        cat <<-USAGE
+$NAME $VERSION
+
+Usage:
+  $NAME [options]
+
+Options:
+  --help
+      Show this help text and exit.
+
+  --version
+      Show version information and exit.
+
+  -c <SECONDS>, --clear <SECONDS>, --clear=<SECONDS>
```

Comment:
I tried the `--clear=<SECONDS>` but got the reply that `the argument --clear=2 is not recognized`. the other 2 work fine.
(same for `--state-path=<PATH>`)

Reply:
Strange. What version of `getopt` do you have?

```
› getopt --version
getopt from util-linux 2.33
```

Does the manpage show the `=` variants for you?

>  A long option normally begins with '--' followed by the long option name.  If the option has a required argument, it may be  written  directly  after the long option name, separated by '=', or as the next argument (i.e., separated by whitespace on the command line).

Reply:
I see now. It's because of the merge issue. Let me push that and you can try again!

Reply:
I think the gnu getopt didn't come with this PR too, can't seem to find it in this diff 😛